gensrc/thrift/Types.thrift

```diff
@@ -447,3 +447,11 @@ enum TCompressionType {
     BZIP2 = 10;
     LZO = 11; // Deprecated
 }
+
+const i64 MAX_VARCHAR_LENGTH = 1048576;
+const i64 MAX_CHAR_LENGTH = 255;
+const i64 MAX_CHAR_INLINE_LENGTH = 128;
+const i64 DEFAULT_STRING_LENGTH = 65533;
+const i64 DEFAULT_HLL_LENGTH = 16385;
+const i64 DEFAULT_BITMAP_LENGTH = 128;
+const i64 DEFAULT_JSON_LENGTH = 128;
```

Comment:
It is very confusing that BITMAP_LENGTH/JSON_LENGTH is 128. I think this PR is your first step to refactor type length.
Later on you will give memory-size and storage-size to each kind of type?

Reply:
Current number `128` is from existing code. The `128` means a default size for memory estimation.